.travis.yml

```diff
@@ -0,0 +1,6 @@
+sudo: false
+language: node_js
+node_js:
+  - "7.7.1"
```

Comment:
Should we bump this to Node v8? That's what I've been doing for all my recent projects, since v8.9.x is LTS.

Reply:
7.7 is also what we define in `package.json`. Isn't this just what we're telling Travis to build with? I'm fine either way.

Reply:
> Isn't this just what we're telling Travis to build with?

Yeah - so I guess _both_ should be bumped to Node 8. That might be out of the scope of this PR, so 🤷‍♂️